frontend/src/lib/components/appLogo/AppLogo.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+  import {page} from '$app/stores';
+  import type {AppLogoProps} from './AppLogo.type';
+
+  type $$Props = AppLogoProps;
+  export let alt: $$Props['alt'] = $page.data.appLabels.viewTexts.publisherName;
+  export let inverse: $$Props['inverse'] = false;
+  export let size: $$Props['size'] = 'md';
+
+  // Create class names
+  let classes: string;
+
+  // Predefined sizes
+  switch (size) {
+    case 'sm':
+      classes = 'h-20 pt-2';
+      break;
+    case 'lg':
+      classes = 'h-32 pt-6';
+      break;
+    default:
+      classes = 'h-24 pt-4';
+  }
+
+  // Merge classes into restProps
+  $$restProps.class = `${classes} ${$$restProps.class ?? ''}`;
+
+  // TODO: Use logo files defined in Strapi.
+  // TODO: Possibly allow passing props directly to the img elements
+</script>
+
+<!--
+@component
+Used to show the application's logo. The logo colour changes dynamically based
+on whether the light or dark mode is active.
+
+### Properties
+
+- See `AppLogo.type.ts` or code completion info.
```

Comment:
if we think of this as a library. it is not really helpful to say that you should see the internal files of the library 😅

Reply:
I'll add the descriptions from the type file like in the other components :smile: